service/cognitosync/api_op_ListDatasets.go

```diff
@@ -72,7 +72,7 @@ type ListDatasetsInput struct {
 	IdentityPoolId *string
 
 	// The maximum number of results to be returned.
-	MaxResults int32
+	MaxResults *int32
```

Comment:
idk why things like this are here. this is breaking. lets discuss this on monday

Reply:
These were expected for `cognito-sync`.